go.mod

```diff
@@ -47,6 +47,7 @@ require (
 	github.com/google/martian/v3 v3.0.0
 	github.com/google/nftables v0.0.0-20220808154552-2eca00135732
 	github.com/gopacket/gopacket v1.1.1
+	github.com/grpc-ecosystem/go-grpc-middleware v1.3.0
```

Comment:
Is this package necessary? The v2 is not better?

Reply:
this was imported by the IDE. I fixed the imports and use the same version now